src/calculateRentalCost.js

```diff
@@ -5,6 +5,22 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+  let total;
+
+  if (days < 3) {
+    total = price * days;
+  }
+
+  if (days >= 3 && days < 6) {
+    total = price * days - 20;
+  }
+
+  if (days >= 7) {
```

Comment:
The conditions for applying discounts overlap. The condition `days >= 3 && days < 6` and `days >= 7` do not cover the case where `days` is exactly 6. You should adjust the conditions to ensure that all possible values of `days` are covered without overlap or gaps. Consider changing the second condition to `days >= 3 && days < 7` to cover the case where `days` is exactly 6.